plugin/trino-phoenix/src/main/java/io/trino/plugin/phoenix/ConfigurationInstantiator.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.phoenix;
+
+import org.apache.hadoop.conf.Configuration;
+import org.gaul.modernizer_maven_annotations.SuppressModernizer;
+
+import static com.google.common.base.Preconditions.checkState;
+
+final class ConfigurationInstantiator
+{
+    private ConfigurationInstantiator() {}
+
+    public static Configuration newEmptyConfiguration()
```

Comment:
Why do we need this copy? Can'w we use trino-hadoop-toolkit?

Reply:
Phoenix and Phoenix5 have different hadoop dependencies. I didn't want to mess with Phoenixes' class paths, so i added a copy instead.